config.analysis

```diff
@@ -41,6 +48,9 @@ climo_yr1 = 6
 climo_yr2 = 10
 # the offset year to be added to simulation years
 yr_offset = 1849
+# start and end date for the analysis
+analysis_start_date = 0001-01-01_00:00:00
+analysis_end_date = 9999-01-01_00:00:00
```

Comment:
Down the road, one option would be a quick pre-processing step (presumably one would be needed for each stream we use) that would go through all files from that stream, read all xtime values, and cache a list of only those files with xtime within the desired range.  Too much for this PR, though.